x-pack/plugins/security_solution/public/hosts/pages/hosts.tsx

```diff
@@ -147,7 +157,11 @@ const HostsComponent = () => {
   return (
     <>
       {indicesExist ? (
-        <div onKeyDown={onKeyDown} ref={containerElement}>
+        <StyledFullHeightContainer
+          onKeyDown={onKeyDown}
+          ref={containerElement}
+          style={{ height: '100%' }}
```

Comment:
nit -> I am wondering if the `height: '100%'` can go with the `styled.div`

Reply:
oh, thought I removed that. Thanks. Was so close with CI 😂